Sources/TuistKit/Services/Cache/CacheWarmService.swift

```diff
@@ -28,7 +29,8 @@ final class CacheWarmService {
         } else {
             cacheController = cacheControllerFactory.makeForSimulatorFramework(contentHasher: contentHasher)
         }
-        try cacheController.cache(path: path)
+        let flavor = try cacheFlavor(named: flavor, from: config)
```

Comment:
@mollyIV bear in mind that we need to extend `CacheContentHasher`'s hashing logic to include the values from the flavor configuration. Otherwise 2 different flavors, which yield different binaries, will have the same hash.

Reply:
Any suggestions where we could add it?